modules/gke-service-account/main.tf

```diff
@@ -22,3 +22,9 @@ resource "google_project_iam_member" "service_account-monitoring_viewer" {
   role    = "roles/monitoring.viewer"
   member  = "serviceAccount:${google_service_account.service_account.email}"
 }
+
```

Comment:
Let's add a comment here that links the related issue and says something along the lines of 'Workaround for a GKE bug that impacts regional clusters due to how they send their identity to the backend.'

Reply:
To my understanding this is not a Bug but working as intended (and only the documentation is missing), since the Service Account of the cluster does actually write metadata.
Please correct me if my understanding is wrong!

Reply:
Actually, I agree with you, I've re-read the linked issue and realized the above needs to be added in addition vs being a bug workaround. Thanks, lets run with this!